ctms/acoustic_service.py

```diff
@@ -0,0 +1,273 @@
+import logging
+import os
+import re
+from functools import lru_cache
+
+from django.utils.encoding import force_bytes
+from lxml import etree
+from silverpop.api import Silverpop, SilverpopResponseException
+
+from ctms import config
+from ctms.schemas import ContactSchema
+
+logger = logging.getLogger(__name__)
+
+
+@lru_cache()
+def get_settings():
+    return config.Settings()
```

Comment:
used below for config env vars:

https://github.com/mozilla-it/ctms-api/pull/188/files#diff-4b3cc193cf9732b1c5ea7f0e45a90c6c9234db1217ae97bc831babc0c505eb7dR267

https://github.com/mozilla-it/ctms-api/pull/188/files#diff-4b3cc193cf9732b1c5ea7f0e45a90c6c9234db1217ae97bc831babc0c505eb7dR260